client/src/main/java/io/pravega/client/ClientFactory.java

```diff
@@ -152,6 +156,36 @@ static ClientFactory withScope(String scope, ClientConfig config) {
     @Beta
     BatchClient createBatchClient();
 
+    /**
+     * Creates a new {@link TableWriter} for the given Table Name, using the given Key and Value Serializers and Config.
```

Comment:
To clarify to the user, I suggest we say that the table must be  created first via `TableManager`.

Reply:
The `createEventWriter` does not have such verbiage. If you want this change, I suggest we file another issue to fix the Javadoc for the client API.